frontend/app/controllers/Event.scala

```diff
@@ -134,13 +134,12 @@ class Event(
   }
 
   def buy(id: String): Action[AnyContent] = eventbriteService.getEvent(id) match {
-      case Some(event@(_: GuLiveEvent)) =>
-        BuyAction(id).async { implicit request =>
-          if (event.isBookableByTier(request.subscriber.subscription.plan.tier))
-            redirectSignedInMemberToEventbrite(event)
-          else suggestUserUpgrades
```

Comment:
are the suggestUserUpgrades and isBookableByTier functions still needed?

Reply:
We can delete the former, the latter is used in DestinationService.scala for the deeplinked event purchase page.

Reply:
Yep, sorry I just pulled your branch and found that out (hence the edited comment!)